dockstore-client/src/main/java/io/dockstore/client/cli/nested/WesCommandParser.java

```diff
@@ -55,12 +57,10 @@ public boolean isHelp() {
     public static class CommandLaunch extends WesMain {
         @Parameter(names = "--entry", description = "Complete workflow path in Dockstore (ex. NCI-GDC/gdc-dnaseq-cwl/GDC_DNASeq:master)", required = true)
         private String entry;
-        @Parameter(names = "--json", description = "Parameters to the entry in Dockstore, one map for one run, an array of maps for multiple runs")
+        @Parameter(names = "--json", description = "JSON file describing which attached file contains input parameters.")
         private String json;
-        @Parameter(names = "--yaml", description = "Parameters to the entry in Dockstore, one map for one run, an array of maps for multiple runs")
-        private String yaml;
-        @Parameter(names = "--uuid", description = "Allows you to specify a uuid for 3rd party notifications")
-        private String uuid;
+        @Parameter(names = {"--attach", "-a"}, description = "A list of paths to files that should be included in the WES request", variableArity = true)
```

Comment:
How are multiple files separated? By comma, space, or something else? I actually don't know :), so it would be good to add it to the description.

Reply:
Files are separated by spaces, or the parameter can be used multiple times. The following are equivalent:
1. `-a <file1> <file2> <file3>`
2. `-a <file1> -a <file2> -a <file3>`

Updated the parameter description and help command: https://github.com/dockstore/dockstore-cli/pull/100/commits/3e058383cb1567468bdcbb089b54981fd24095e9